services/gateway/pkg/config/config.go

```diff
@@ -32,15 +32,15 @@ type Config struct {
 
 	UsersEndpoint             string `yaml:"users_endpoint" env:"GATEWAY_USERS_ENDPOINT" desc:"The USERS API endpoint."`
 	GroupsEndpoint            string `yaml:"groups_endpoint" env:"GATEWAY_GROUPS_ENDPOINT" desc:"The GROUPS API endpoint."`
-	PermissionsEndpoint       string `yaml:"permissions_endpoint" env:"GATEWAY_PERMISSIONS_ENDPOINT" desc:"The permission API endpoint."`
+	PermissionsEndpoint       string `yaml:"permissions_endpoint" env:"GATEWAY_PERMISSIONS_ENDPOINT" desc:"The SETTINGS API endpoint."`
```

Comment:
SETTINGS?

Reply:
Yes, settings. Clarified with @wkloucek, this is a bad naming (history) but points to the settings service, which you also can see when looking to the admin docu at the default port used and double check this with the port list in the main service description -> settings...